### PR TITLE
Add instructions for adding root domain, with www. redirect to Cloudflare

### PR DESCRIPTION
**Pull request by Tony-Hunter**
Include detailed instructions on how to get both the root domain, and the www. subdomain redirecting to the route domain, with full SSL enabled.

### CODE REVIEW

File: src/docs/guides/public-networking.md [hunk 1]
```diff
@@ -140,6 +140,43 @@ The type of record to create is entirely dependent on your DNS provider.  Here a
...
+    - `Target` → the `value` field from Railway.
+    - `Proxy status` → `on`, should display an orange cloud.
+    - Note: Due to domain flattening, `Name` will automatically update to your root domain (e.g., `mydomain.com`).
+3. Add another `CNAME` DNS record to Cloudflare:
+    - `Name` → `www`.
+    - `Target` → `@`
+    - `Proxy status:` → on, should display an orange cloud.
+    - Note: Cloudflare will automatically change the `Target` value to your root domain.
+4. Enable Full SSL/TLS encryption in Cloudflare:
+    1. Go to your domain on Cloudflare.
```
Comment (brody192) [inline src/docs/guides/public-networking.md:159]: You switched from bullet points to numbers, please keep the bullet points throughout all sub-steps for consistency.

File: src/docs/guides/public-networking.md [hunk 2]
```diff
@@ -140,6 +140,43 @@ The type of record to create is entirely dependent on your DNS provider.  Here a
...
+    2. Navigate to `Bulk Redirects`.
+    3. Click `Create Bulk Redirect List`.
+    4. Give it a name, e.g., `www-redirect`.
+    5. Click `Or, manually add URL redirects`.
+    6. Add a `Source URL`: `https://www.mydomain.com`.
+    7. Add `Target URL`: `https://mydomain.com` with status `301`.
+    8. Tick all the parameter options: (`Preserve query string`, `Include subdomains`, `Subpath matching`, `Preserve path suffix`)
+    9. Click `Next`, then `Save and Deploy`.
+
+> **Note:** DNS changes may take some time to propagate. You may want to refresh your DNS cache by using commands like `ipconfig /flushdns` on Windows or `dscacheutil -flushcache` on macOS. Testing the URLs in an incognito window can also help verify changes.
```
Comment (brody192) [inline src/docs/guides/public-networking.md:178]: I don't think this should be a quote, remove the >, and it's good.

File: src/docs/guides/public-networking.md [hunk 3]
```diff
@@ -140,6 +140,43 @@ The type of record to create is entirely dependent on your DNS provider.  Here a
 
 If your DNS provider doesn't support CNAME Flattening or dynamic ALIAS records, you can also change your domain's nameservers to point to Cloudflare's nameservers. This will allow you to use a CNAME record for the root domain. Follow the instructions listed on Cloudflare's documentation to <a href="https://developers.cloudflare.com/dns/zone-setups/full-setup/setup/" target="_blank">change your nameservers</a>.
 
+**Adding a root domain with www. subdomain to Cloudflare**
```
Comment (brody192) [inline src/docs/guides/public-networking.md:143]: This should be a header.

**Comment by Tony-Hunter** [comment]
@brody192, addressed feedback

**Comment by brody192** [comment]
Thanks for the updates, LGTM now!

**Pull request merged**